### PR TITLE
Add helm chart configuration to readme

### LINKED ISSUE #2103: The config options for Helm chart need to be listed in a table
The readme here https://github.com/dapr/dapr/tree/master/charts/dapr needs to include all the options that can be provided to the Helm chart with an example. For example like this https://github.com/grafana/helm-charts/tree/main/charts/grafana

For the example show an example like this

helm install dapr dapr/dapr --namespace dapr-system --set global.ha.enabled=true

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Add helm chart configuration to readme

## Issue reference

Closes #2103

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: charts/dapr/README.md [hunk 1]
```diff
@@ -46,3 +46,44 @@ To uninstall/delete the `dapr` release:
 ```
 helm uninstall dapr -n dapr-system
 ```
+
+## Configuration
+
+| Parameter                                 | Description                                                             | Default                 |
+|-------------------------------------------|-------------------------------------------------------------------------|-------------------------|
+| `global.registry`                         | Global Dapr docker image registry                                       | `docker.io/daprio`      |
+| `global.tag`                              | Global Dapr docker image version tag                                    | `0.10.0`                |
```
Comment (mukundansundar) [inline charts/dapr/README.md:55]: default version is the version of the helm chart?

Comment (youngbupark) [reply charts/dapr/README.md]: it is runtime version. I changed to 0.11.0 for now.

**Review by yaron2 — APPROVED** [review]
lgtm

**Comment by youngbupark** [comment]
I missed adding upgrade and ha setup example.. I will update it.

**Review by yaron2 — APPROVED** [review]

File: charts/dapr/README.md [hunk 2]
```diff
@@ -40,13 +40,18 @@ Once the chart installation is done, verify the Dapr operator pods are running i
...
+## Uninstall the Chart
 
 To uninstall/delete the `dapr` release:
 ```
 helm uninstall dapr -n dapr-system
 ```
 
+## Upgrade the charts
+
+Before upgrading Dapr, ensure that existing certs in the cluster need to be exported. Follow the upgrade instruction in [Upgrading Dapr with Helm](https://github.com/dapr/docs/blob/master/howto/deploy-k8s-prod/README.md#upgrading-dapr-with-helm).
```
Comment (mchmarny) [inline charts/dapr/README.md:52]: > Before upgrading Dapr, make sure to export existing certs. Follow the upgrade instruction in [Upgrading Dapr with Helm](https://github.com/dapr/docs/blob/master/howto/deploy-k8s-prod/README.md#upgrading-dapr-with-helm).

**Review by mchmarny — CHANGES_REQUESTED** [review]
one comment

File: charts/dapr/README.md (hunk 2 above)
Comment (youngbupark) [reply charts/dapr/README.md]: fixed.

**Review by mchmarny — APPROVED** [review]
/lgtm

**Pull request merged**